### PR TITLE
clarified per app

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/osv2-faq.adoc [hunk 1]
```diff
@@ -45,7 +45,7 @@ When other Mule apps need access to your app's object stores, use the https://an
 
 == What are the limits on an object store?
 
-There’s no limit of the total size of an object store, but each value is limited to 10 MB. The free version also limits usage to 10 transactions per second. Premium add-on users get up to 100 TPS.
+There’s no limit of the total size of an object store, but each value is limited to 10 MB. The free version also limits usage to 10 transactions per second (TPS). Premium add-on users get up to 100 TPS per app.
```
Comment (prafulrana) [inline modules/ROOT/pages/osv2-faq.adoc]: should we say "application" instead of "app"?

**Review by prafulrana — APPROVED** [review]

File: modules/ROOT/pages/osv2-faq.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-faq.adoc]: Salesforce/MuleSoft style is to use "app" (so slowly changing...)

Comment (Cristian-Venticinque) [inline modules/ROOT/pages/osv2-faq.adoc]: "There's no limit of..." 
I'm not completely sure but "limit of the..." makes some noise to me. I think this should be either limit **on**, or limit **for**. Can you please confirm with Kim?

**Review by Cristian-Venticinque — APPROVED** [review]

File: modules/ROOT/pages/osv2-faq.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-faq.adoc]: Yep. You're right. Thanks for your eagle eye.

**Pull request merged**